ui/apps/everest/src/components/confirm-form-dialog/confirm-form-dialog.tsx

```diff
@@ -61,15 +61,26 @@ export const ConfirmFormDialog = ({
     submitting={submitting}
     {...props}
   >
-    <IrreversibleAction />
-    <DialogContent sx={{ px: 1 }}>{dialogContent}</DialogContent>
-    <TextInput
-      name={ConfirmFormDialogFields.confirmInput}
-      label={inputLabel}
-      labelProps={{ sx: { mt: 0 } }}
-      textFieldProps={{
-        placeholder: inputPlaceholder,
-      }}
-    />
+    {({ setValue, trigger }) => (
+      <>
+        <IrreversibleAction />
+        <DialogContent sx={{ px: 1 }}>{dialogContent}</DialogContent>
+        <TextInput
+          name={ConfirmFormDialogFields.confirmInput}
+          label={inputLabel}
+          labelProps={{ sx: { mt: 0 } }}
+          textFieldProps={{
+            onChange: (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
https://stackoverflow.com/questions/77134910/how-can-i-remove-all-whitespace-in-zod it's more correct to do it via zod

Reply:
Ok, i'll fix it now